pkg/konfluxgen/integration-test-scenario.template.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: appstudio.redhat.com/v1beta2
+kind: IntegrationTestScenario
+metadata:
+  name: {{{ truncate ( sanitize .ApplicationName ) }}}-enterprise-contract
+  namespace: ocp-serverless-tenant
+spec:
+  params: 
+    - name: POLICY_CONFIGURATION
+      value: rhtap-releng-tenant/tmp-onboard-policy
+  application: {{{ .ApplicationName }}}
```

Comment:
```suggestion
  application: {{{ truncate ( sanitize .ApplicationName ) }}}
```

Reply:
This is because `spec.application` is a reference to an Application name and the name is a truncated and sanitized name https://github.com/openshift-knative/hack/blob/7587057e3c32be19f860767c26b3cbf20b2aea25/pkg/konfluxgen/application.template.yaml#L4